src/Nri/Ui/Table/V8.elm

```diff
@@ -117,27 +119,86 @@ rowHeader options =
 
 
 
+-- Model
+
+
+type alias Config =
+    { css : List Style, alternatingRowColors : Bool }
+
+
+{-| Attribute to configure the table
+-}
+type Attr
+    = Attr (Config -> Config)
+
+
+defaultConfig : Config
+defaultConfig =
+    { css = [], alternatingRowColors = True }
+
+
+{-| Add a CSS style to the table
+-}
+css : List Style -> Attr
+css styles =
+    Attr <|
+        \config ->
+            { config | css = styles ++ config.css }
```

Comment:
can we make add the new customizations at the end so they take precedence over default styles? that's how `additionalStyles` works in V7, I think.